staging/vhost-device-can/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "vhost-device-can"
+version = "0.1.0"
+authors = ["Timos Ampelikiotis <t.ampelikiotis@virtualopensystems.com>"]
+description = "vhost can backend device"
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["can", "vhost", "virt", "backend"]
+license = "Apache-2.0 OR BSD-3-Clause"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+xen = ["vm-memory/xen", "vhost/xen", "vhost-user-backend/xen"]
+
+[dependencies]
+clap = { version = "4.4",  features = ["derive"] }
+env_logger = "0.11"
+log = "0.4"
+thiserror = "1.0"
+queues = "1.0.2"
+socketcan = { git = "https://github.com/socketcan-rs/socketcan-rs.git", rev = "f004ee91e142a37fea36c5d719a57852c7076e87" }
```

Comment:
Can you add a TODO here to remember us to fix it at `v3.4.0` when it will be release?
We will need that before doing a release on crates.io, but since this device is still in staging, we have time ;-)

Reply:
Please also add the reason Matias described in the TODO comment, so we remember why we need that version.

Reply:
Thanks @stefano-garzarella! I have added both issues into the `Cargo.toml` file

Reply:
great, thanks!